manifests/s/Swift/Toolchain/5.6.1/Swift.Toolchain.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-22567-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Swift.Toolchain
+PackageVersion: "5.6.1"
+PackageLocale: en-US
+Publisher: dt.compnerd.org
```

```suggestion
Publisher: swift.org
```
to match Control Panel.